.github/workflows/release.yml

```
@@ -0,0 +1,76 @@
+        tag_name: ${{ github.ref_name || github.event.inputs.version }}
+        name: Release ${{ github.ref_name || github.event.inputs.version }}
+        draft: false
+        prerelease: false
+        body: |
+          ## Installation
+          
+          Install directly from this release:
+          ```bash
+          pip install https://github.com/atomicworkhq/crawl4ai/releases/download/${{ github.ref_name || github.event.inputs.version }}/${{ steps.get-files.outputs.wheel_name }}
+          ```
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)